Classes/Controller/BackendController.php

```diff
@@ -118,7 +118,7 @@ public function buildFormAction($view = 'BuildForm') {
 	 * @return void
 	 */
 	public function buildAction($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs, $dry, $verbose) {
-		$generator = $this->extensionService->buildProviderExtensionGenerator($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs, $dry, $verbose);
+		$generator = $this->extensionService->buildProviderExtensionGenerator($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs);
```

Comment:
Both "dry" and "verbose" are still valid arguments. Travis, git etc. arguments are removed.